tests/test_hilbert_curve.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import pandas as pd
+from hilbertcurve.hilbertcurve import HilbertCurve
+from dask_geopandas.hilbert_distance import _continuous_to_discrete_coords
+from dask_geopandas import from_geopandas
+from .test_core import geoseries_points, geoseries_lines, geoseries_polygons
+
+
+@pytest.fixture(params=[geoseries_points, geoseries_lines, geoseries_polygons])
+def geom_types(request):
+    return request.getfixturevalue(request.param)
+
+
+@pytest.fixture
```

Comment:
```suggestion
```

This is a test, not a fixture. Pytest doesn't run it now.

Reply:
woops, that wasn't supposed to be there. fixed in c8fbd7f